test/ConventionalCommitUtilities.js

```diff
@@ -0,0 +1,114 @@
+import assert from "assert";
+
+import dedent from "dedent";
+import assertStubbedCalls from "./_assertStubbedCalls";
+import ConventionalCommitUtilities from "../src/ConventionalCommitUtilities";
+import ChildProcessUtilities from "../src/ChildProcessUtilities";
+import FileSystemUtilities from "../src/FileSystemUtilities";
+
+describe("ConventionalCommitUtilities", () => {
+  describe(".recommendVersion()", () => {
+    it("should invoke conventional-changelog-recommended bump to fetch next version", () => {
+      assertStubbedCalls([
+        [ChildProcessUtilities, "execSync", {}, [
+          {
+            args: [require.resolve("conventional-recommended-bump/cli.js") + " -l bar --commit-path=/foo/bar -p angular"],
+            returns: "major"
+          }
+        ]],
+      ]);
+
+      const recommendedVersion = ConventionalCommitUtilities.recommendVersion({
+        name: "bar",
+        version: "1.0.0",
+        location: "/foo/bar"
+      });
+
+      assert.equal(recommendedVersion, "2.0.0");
+    });
+  });
+
+  describe(".updateChangelog()", () => {
+    it("should populate initial CHANGELOG.md if it does not exist", () => {
+      assertStubbedCalls([
+        [FileSystemUtilities, "existsSync", {}, [
+          {
+            args: ["/foo/bar/CHANGELOG.md"],
+            returns: false
+          }
+        ]],
+        [ChildProcessUtilities, "execSync", {}, [
+          {
+            args: [`${require.resolve("conventional-changelog-cli/cli.js")} -l bar --commit-path=/foo/bar --pkg=/foo/bar/package.json -p angular`],
+            returns: "<a name='change' />feat: I should be placed in the CHANGELOG"
+          }
+        ]],
+        [FileSystemUtilities, "writeFileSync", {}, [
+          {
+            args: ["/foo/bar/CHANGELOG.md", dedent(`
```

Comment:
It's implicitly called behind the scenes when you use `path.join()` or `path.resolve()`, too, though for this purpose a direct `normalize` call seems appropriate.